packages/services/server/src/graphql-handler.ts

```diff
@@ -210,3 +232,42 @@ export const graphqlHandler = (options: GraphQLHandlerOptions): RouteHandlerMeth
     );
   };
 };
+
+if (!process.env['SUPERTOKENS_CONNECTION_URI']) {
+  throw new Error('Missing SUPERTOKENS_CONNECTION_URI env variable.');
+}
+
+if (!process.env['SUPERTOKENS_API_KEY']) {
+  throw new Error('Missing SUPERTOKENS_API_KEY env variable.');
+}
+
+const connectionUri = process.env['SUPERTOKENS_CONNECTION_URI'];
+const apiKey = process.env['SUPERTOKENS_API_KEY'];
+
+/**
+ * Verify whether a SuperTokens access token session is valid.
+ * https://app.swaggerhub.com/apis/supertokens/CDI/2.15.1#/Session%20Recipe/verifySession
+ */
+async function verifySuperTokensSession(accessToken: string): Promise<SuperTokenSession> {
+  const response = await fetch(connectionUri + '/recipe/session/verify', {
```

Comment:
do you want to experiment with mesh as SDK for this swagger? or it's an overkill?